frontend/public/locales/fr/translation.json

```diff
@@ -3241,9 +3241,9 @@
   "validation.missing.value": "La valeur {{0}} est requise",
   "Value": "Valeur",
   "Value must be a valid IPv4 CIDR.": "La valeur doit être un CIDR IPv4 valide.",
-  "Value must be in <namespace>/<name> format.": "La valeur doit être dans<namespace> /<name> format.",
-  "Value must be in <user>@<domain> format.": "La valeur doit être dans<user> @<domain> format.",
-  "Value must be integer greater than": "La valeur doit être un entier supérieur à{{0}} .",
+  "Value must be in <namespace>/<name> format.": "La valeur doit être dans <namespace>/<name> format.",
+  "Value must be in <user>@<domain> format.": "La valeur doit être dans <user>@<domain> format.",
+  "Value must be integer greater than": "La valeur doit être un entier supérieur à {{0}} .",
```

Comment:
Missing spaces. Also, technically, the words inside the `<` and `>` characters could be translated, as these are placeholder variables that the user will fill in with appropriate values.